core/templates/dev/head/domain/topic/SubtopicPageContentsObjectFactorySpec.ts

```diff
@@ -48,7 +48,8 @@ describe('Subtopic page contents object factory', () => {
           en: {
             filename: 'test.mp3',
             file_size_bytes: 100,
-            needs_update: false
+            needs_update: false,
+            duration_secs: 0.2
```

Comment:
Isn't the `subtopic_page_contents` object modified here? If so, you would need to write a migration for it. The function to handle the migration is present in subtopic_page_services, but the `CURRENT_SUBTOPIC_PAGE_CONTENTS_SCHEMA_VERSION ` is still v1, so this would be the first one.

Reply:
@aks681, I think we had discussed this earlier over [here](https://github.com/oppia/oppia/pull/8203#discussion_r358666272)
![image](https://user-images.githubusercontent.com/16653571/75182667-e2b4c580-5766-11ea-8de1-bc5950be1c43.png)

Reply:
Unfortunately, this PR has taken too long to get in and the situation has changed.

Reply:
Oh, ok. So, there are no blank objects stored if no audio is recorded, right? Since, we do have subtopics on the test server, albeit with no audio recorded. So as long as they can be loaded fine with this PR, it's ok.

Reply:
@aks681, Yeah the recorded_voicoever structure of the subtopic is `{}` for all the subtopic as there isn't any functionality to upload audio for the subtopic. (So the Voiceover object schema isn't available in any of the new structure projects.)

Reply:
Ok, in that case, I think it is fine!